VideoWeb/VideoWeb/ClientApp/src/app/services/video-filter.service.ts

```diff
@@ -168,7 +168,7 @@ export class VideoFilterService {
 
     doesSupportVideoFiltering() {
         const allowedBrowser = !this.deviceTypeService.getBrowserName().includes(browsers.Safari);
-        return this._enableVideoFilters && allowedBrowser && !this.deviceTypeService.isTablet();
+        return this._enableVideoFilters && allowedBrowser && !this.deviceTypeService.isDesktop();
```

Comment:
Need to get rid of the not otherwise this will only support mobile and tablets
```suggestion
        return this._enableVideoFilters && allowedBrowser && this.deviceTypeService.isDesktop();
```